src/config/types.ts

```diff
@@ -15,13 +15,13 @@ export interface BuildDiffOptsConfig {
     ignoreCaret: boolean;
 }
 
-export interface AssertViewOptsConfig {
+export interface AssertViewOpts {
```

Comment:
Just rename, make it optional and add fields from `src/browser/commands/types.ts`